pkg/kafka/consumer/config.go

```diff
@@ -52,24 +49,27 @@ type Configuration struct {
 
 // NewConsumer creates a new kafka consumer
 func (c *Configuration) NewConsumer(logger *zap.Logger) (Consumer, error) {
-	saramaConfig := cluster.NewConfig()
-	saramaConfig.Group.Mode = cluster.ConsumerModePartitions
+	saramaConfig := sarama.NewConfig()
 	saramaConfig.ClientID = c.ClientID
 	saramaConfig.RackID = c.RackID
 	if len(c.ProtocolVersion) > 0 {
 		ver, err := sarama.ParseKafkaVersion(c.ProtocolVersion)
 		if err != nil {
 			return nil, err
 		}
-		saramaConfig.Config.Version = ver
+		saramaConfig.Version = ver
 	}
-	if err := c.AuthenticationConfig.SetConfiguration(&saramaConfig.Config, logger); err != nil {
+	if err := c.AuthenticationConfig.SetConfiguration(saramaConfig, logger); err != nil {
 		return nil, err
 	}
 	// cluster.NewConfig() uses sarama.NewConfig() to create the config.
```

Comment:
looks like this comment may no longer be applicable

Reply:
ok, I will remove
---
Deprecation warning: Consumer.Offsets.CommitInterval exists for historical compatibility and should not be used. Please use Consumer.Offsets.AutoCommit, the current value will be ignored

<img width="832" alt="image" src="https://user-images.githubusercontent.com/8416556/226918531-b4b6085a-1deb-42ca-af1e-a16c7cc973e3.png">